core/common/src/test/java/alluxio/EmbeddedChannels.java

```diff
@@ -50,8 +50,6 @@ public EmbeddedEmptyCtorChannel() {
       ChannelPipeline p = super.pipeline();
       // Remove the dummy handler.
       p.removeFirst();
-      // Remove the LastInboundHandler in super;
-      p.removeLast();
```

Comment:
why is this no longer needed?

Reply:
We were using a hack to get around some limitations before.

https://alluxio.atlassian.net/browse/ALLUXIO-3066